### PR TITLE
WEBEU2 - Redux-Todo - Sulaiman Abidemi 🇳🇬

### PR DESCRIPTION
**Pull request by abidex4yemi**
### What does this commit do ?

* Create basic Todo app using Redux as a state container

### CODE REVIEW

File: todo/src/components/App/App.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import styled from 'styled-components';
+import TodoContainer from '../Todo/TodoContainer';
+import Form from '../Todo/Form';
+import { store } from '../../state/store/store';
+import './App.css';
+
+const AppStyled = styled.div`
```
Comment (maximesalomon) [inline todo/src/components/App/App.jsx:9]: You might want to add your styled components after your component logic to ease readability

File: todo/src/components/Todo/Form/InputPresentation.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+				name={name}
+				onChange={evt => inputChange(evt.target.name, evt.target.value)}
+				placeholder={placeholder}
+			/>
+			<small>{error || ''}</small>
+		</div>
+	);
+};
+
+InputPresentation.propTypes = {
```
Comment (maximesalomon) [inline todo/src/components/Todo/Form/InputPresentation.jsx:37]: Great use of PropTypes

File: todo/src/state/reducers/index.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from 'redux';
```
Comment (maximesalomon) [inline todo/src/state/reducers/index.js:1]: In this project - there is no need to create another file to handle this. You add complexity to your app architecture.

**Review by maximesalomon — COMMENTED** [review]
Hello Yemi 👋 

Great job on this project 🔥

Are you using Prettier or a code formatter?

**Comment by abidex4yemi** [comment]
> Hello Yemi 👋
> 
> Great job on this project 🔥
> 
> Are you using Prettier or a code formatter?

Yes i'm using Prettier.

File: todo/src/state/reducers/index.js (hunk 3 above)
Comment (abidex4yemi) [reply todo/src/state/reducers/index.js:1]: Yes that's true, I did it that way for experimental purposes just to see how it works. Thanks for your guidance

File: todo/src/components/App/App.jsx (hunk 1 above)
Comment (abidex4yemi) [reply todo/src/components/App/App.jsx:9]: Ok, thanks

File: todo/src/components/Todo/Form/InputPresentation.jsx (hunk 2 above)
Comment (abidex4yemi) [reply todo/src/components/Todo/Form/InputPresentation.jsx:37]: Thank you.

**Pull request merged**